### PR TITLE
add doc tests for ambient getting started guide

### LINKED ISSUE #12802: Follow up work for get started guide - page needs to be tested
During review of https://github.com/istio/istio.io/pull/12783, we need to add a test to test this page.

### PR DESCRIPTION
**Pull request by kfaseela**
Please provide a description for what this PR is for.

Fixes #12802

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/ambient/getting-started/index.md [hunk 1]
```diff
@@ -120,9 +148,9 @@ to the ambient mesh. The best part is that there was no need to restart or redep
 Send some test traffic:
 
 {{< text bash >}}
-$ kubectl exec deploy/sleep -- curl -s http://istio-ingressgateway.istio-system/productpage | head -n10
-$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | head -n10
-$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | head -n10
+$ kubectl exec "$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})" -c sleep -- curl -s http://istio-ingressgateway.istio-system/productpage | | head -n10
```
Comment (peterj) [inline content/en/docs/ops/ambient/getting-started/index.md:151]: out of curiosity -- why replace the deploy/[deployment] with kubectl get pod?

Comment (kfaseela) [reply content/en/docs/ops/ambient/getting-started/index.md]: sorry, it is not needed...was just experimenting

**Comment by kfaseela** [comment]
@linsun need to wait for the below two PRs to be pulled into istio.io to do a retest of this test PR. Hoping it will be available in the Monday sync. 

https://github.com/istio/istio/pull/43640
https://github.com/istio/istio/pull/43749

File: content/en/docs/ops/ambient/getting-started/index.md [hunk 2]
```diff
@@ -69,6 +69,24 @@ Follow these steps to get started with ambient:
     ✔ Installation complete
     {{< /text >}}
 
+1.  Verify the installed components using the below commands:
```
Comment (frankbu) [inline content/en/docs/ops/ambient/getting-started/index.md:72]:
```suggestion
1.  Verify the installed components using the following commands:
```

**Review by frankbu — APPROVED** [review]
LGTM

**Comment by ericvn** [comment]
Thanks.

File: content/en/docs/ops/ambient/getting-started/test.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+snip_l7_authorization_policy_5
+_verify_contains snip_l7_authorization_policy_6 "$snip_l7_authorization_policy_6_out"
+_verify_contains snip_l7_authorization_policy_7 "$snip_l7_authorization_policy_7_out"
+_verify_contains snip_l7_authorization_policy_8 "$snip_l7_authorization_policy_8_out"
+
+snip_control_traffic_control_1
+_verify_contains snip_control_traffic_control_1 "waypoint default/bookinfo-reviews applied"
+
+snip_control_traffic_control_2
+# snip_control_traffic_control_3
```
Comment (frankbu) [inline content/en/docs/ops/ambient/getting-started/test.sh:68]: Just realized that there is no verification of this last step in the doc. I would say it's not worth validating the 10-90% traffic-split, but an easier check to make sure that things aren't broken would be good. Maybe like this:
```
_verify_lines snip_control_traffic_control_3 "
+ reviews-v1
+ reviews-v2
"
```

Comment (kfaseela) [reply content/en/docs/ops/ambient/getting-started/test.sh:68]: Added in https://github.com/istio/istio.io/pull/12850

**Pull request merged**